templates/includes/sidebar.html

```diff
@@ -1,4 +1,13 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>User: {{ user.username }}</li>
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/227931802-c2290fad-b7bf-4167-be7e-8e6dc3d05c51.png)
fix this